test/fixtures/transfers.yml

```diff
@@ -0,0 +1,20 @@
+# == Schema Information
+#
+# Table name: transfers
+#
+#  id            :integer          not null, primary key
+#  user_id       :integer          not null
+#  department_id :integer          not null
+#  grad_date     :date             not null
+#  created_at    :datetime         not null
+#  updated_at    :datetime         not null
+#
+valid:
+  department: one
+  grad_date: 2020-05-01
+  user: yo
+
+baduser:
+  department_id: 0
```

Comment:
I believe this fixture should use the format of `department: name_of_department_fixture` instead of a specific `department_id`.

Reply:
Ah, yeah - good catch. I'll fix.